src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+/**
+ * Association class representing an addition/deletion of Tags for the person

```

Comment:
A more precise description is `Represents an adding or removing a Tag to/from a Person`.